### ISSUE TITLE
Remove duplicated strings from babel/number-play, babel/number-suite-common, babel/number-compare files

### ISSUE THREAD

**Issue by chrisklus** [phetsims/number-suite-common#27]
I'm wondering if it's okay to copy and paste NP's string files from babel into a `number-compare` directory to be used as the initial translated strings for the publication of this sim. I've started with this for now because it's helpful with testing dynamic locale stuff. If acceptable, I can go through each file and remove the strings that aren't used in this sim (and remove the strings that aren't used in NP from NP's string files). I think the only string that would be missing from the translations of Number Compare would be the title.

**Comment by samreid** [comment]
I asked @jbphet a similar question in https://github.com/phetsims/sound/issues/5

**Comment by chrisklus** [comment]
Thanks @samreid! I read through that issue.

I talked with @jbphet today and since things are working as I have them, and the copied files have translator credits etc, I should be good to proceed as proposed and publish NC with these translations. He mentioned I should ask @oliver-phet to send out an email to ask translators to add a title translation once the sim is published, as that should end up being the only missing string.

**Comment by chrisklus** [comment]
Putting this on hold until I'm sure of the final set of strings needed for Number Play and Number Compare so I can clean up babel files in one go.

**Comment by pixelzoom** [comment]
@chrisklus @marlitas since this issue also involves strings, you might want to add it to the other string-related issues that you're planning to visit on 1/4/2023.

**Comment by pixelzoom** [comment]
Also note that number-suite-common contains many strings that are not in fact "common" and should not be in the string file for that repo. See https://github.com/phetsims/number-suite-common/issues/32.

**Comment by chrisklus** [comment]
Adding blocks-sim-publication label to indicate this should be completed before starting dev testing.

**Comment by chrisklus** [comment]
for example, number play:

remove all keys in babel/number-play/number-play-strings_* that are not in number-play/number-play-strings_en.json

this should be done in number-compare and number-suite-common too.

We are thinking of a node script that creates an array of keys from the `en` string file and then omits keys+values in the translated JSON files. We should be able to iterate through every file in the dir with `fs.readDirSync`.

To test the script before running, we think it would be ideal to create a fake directory in babel like `number-compare-fake` that's a duplicate of `babel/number-compare`.

**Comment by chrisklus** [comment]
We think this can come off hold because the splitting out of string keys has been completed.

**Comment by chrisklus** [comment]
@marlitas and I completed this in the above commits but are going to wait until right before publishing to run the script. Marking on hold again.

**Comment by chrisklus** [comment]
Adding a note for me and @marlitas when we do this work since I know I'm going to forget:

1. In https://github.com/phetsims/number-compare/issues/14, `"isMoreThan"` string key was renamed to `"isGreaterThan"`. The value of that string key was changed before initial publication, so the translated values for it in babel are correct. This mean we should include the string key + value pair. To do this, we need to rename the string key in `babel/number-play` BEFORE we re-copy all of the translated strings from `babel/number-play` to the other two new sub-directories.
2. ~In https://github.com/phetsims/number-suite-common/issues/47, I added a new string key `language`. This is problematic because the original publication of Number Play also had a string key named `language`, but the value is different now. BEFORE copying `babel/number-play` to the other two new sub-directories, the string key + value needs to be deleted from all translated files. This was going to happen automatically, but since I just re-added it to number-suite-common, it would now incorrectly be preserved.~

EDIT: 2 was handled in https://github.com/phetsims/number-compare/issues/27.

**Comment by chrisklus** [comment]
Yesterday @zepumph and I improved the scripts that we started for this issue and @zepumph enumerated how to use them over in https://github.com/phetsims/my-solar-system/issues/87#issuecomment-1449041208.

**Comment by zepumph** [comment]
Ok. All has been done.

* We renamed "isMoreThan" to "isGreaterThan in number play string files.
* Then we ran this script:

```

cd babel
../rosetta/scripts/sh/copy-babel-subdirectory.sh number-play number-compare
../rosetta/scripts/sh/copy-babel-subdirectory.sh number-play number-suite-common
cd ..
node rosetta/scripts/js/remove-translations-without-english-key.js number-compare
node rosetta/scripts/js/remove-translations-without-english-key.js number-suite-common
node rosetta/scripts/js/remove-translations-without-english-key.js number-play 
```

Everything is looking really good over here. We are proceeding to production publication. Closing
